docs/outlook/smart-alerts-onmessagesend-walkthrough.md

```diff
@@ -135,6 +142,88 @@ Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeo
 > - For **SendMode** options available with the `OnMessageSend` and `OnAppointmentSend` events, see [Available SendMode options](/javascript/api/manifest/launchevent#available-sendmode-options).
 > - To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md).
 
+# [Teams Manifest (developer preview)](#tab/jsonmanifest)
+
+1. Open the manifest.json file.
+
+1. Add the following object to the "extensions.runtimes" array. Note the following about this markup:
+
+   - The "minVersion" of the Mailbox requirement set has been set to "1.12" because the table at the top of this article specifies that this is the lowest version of the requirement set that supports the **OnMessageSend** event.
```

Comment:
```suggestion
   - The "minVersion" of the Mailbox requirement set is set to "1.12" because the [supported events table](autolaunch.md#supported-events) specifies that this is the lowest version of the requirement set that supports the `OnMessageSend` event.
```
Proposing use of backticks with the event name to match formatting in events table.